portfolio/src/main/java/com/google/sps/servlets/CommentsServlet.java

```diff
@@ -72,4 +72,26 @@ private String getParameter(HttpServletRequest request, String name) {
     }
     return value;
   }
+
+  private int getMaxCommentsToDisplay(HttpServletRequest request) {
+    String maxCommentsToDisplayString = "3"; //temporary hardcoded value, will get replaced with user input
+
+    // Convert the input to an int.
+    int maxCommentsToDisplay;
+    try {
+      maxCommentsToDisplay = Integer.parseInt(maxCommentsToDisplayString);
+    } catch (NumberFormatException e) {
+      System.err.println("Could not convert to int: " + maxCommentsToDisplayString);
+      return -1;
```

Comment:
Don't return negative one. Right now I think you pass that to the databse which will probably fail. Instead redirect to an error page or throw an Exception

Reply:
Changed so that it is throwing an exception. I am going to have to think about how to best handle all these exceptions (e.g. error page, alert)